src/test.rs

```diff
@@ -107,3 +107,9 @@ fn positive_test_run_pass() {
 
     compiletest::run_tests(&config);
 }
+
+#[test]
+#[should_panic]
+fn panic() {
+    join(|| {}, || panic!("should panic"));
```

Comment:
Oops, I thought I removed that test. Yes it does have undesirable effects on the other tests. I'll look into adding some run-fail tests.